keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -693,9 +693,12 @@ in back-to-back releases.
 
 #### Beta
 - e2e tests pass with containerd, CRI-O, and cri-dockerd
+ - https://github.com/containerd/containerd/pull/9787
+ - https://github.com/cri-o/cri-o/pull/7962
+ - https://github.com/Mirantis/cri-dockerd/pull/370
 
 #### GA
-- (Will be revisited during beta)
+- Two beta releases at least
```

Comment:
Does GA should ensure the RRO feature be widely adopted, iow. at least 2 or 3 more implementations support it?

Reply:
Just having three implementations (containerd, CRI-O, and cri-dockerd) should be enough IMO.
I'm not even aware of other prod-quality CRI runtimes.